src/common/options/global.yaml.in

```diff
@@ -1018,8 +1018,9 @@ options:
   type: str
   level: advanced
   desc: Compression algorithm to use in Messenger when communicating with OSD
-  long_desc: Compression algorithm for connections with OSD in order of preference
-  default: snappy zlib zstd lz4
+  long_desc: Compression algorithm has been set to snappy as the default 
+    algorithm. However, there is flexibility to set the algorithm manually.
```

Comment:
The commit shouldn't remove the original message that was explaining the purpose of the configurable. I would just add that, although the default value is singular, a list (like `snappy zlib zstd` etc.) is acceptable as well.

Reply:
Done!